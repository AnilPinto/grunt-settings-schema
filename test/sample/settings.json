{
    "auth": {
        "sessionTtl": 500,
        "productDataAuthStateRequired": "AuthenticatedVerified"
    }
}