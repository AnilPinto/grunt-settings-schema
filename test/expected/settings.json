{
    "auth": {
        "sessionTtl": 500,
        "productDataAuthStateRequired": "AuthenticatedVerified",
        "stateLevels": {
            "Unauthenticated": 0,
            "Identified": 1,
            "IdentifiedVerified": 2,
            "Authenticated": 3,
            "AuthenticatedVerified": 4
        },
        "authSessionEvents": {
            "end": {
                "authState": "Unauthenticated",
                "uiState": "login"
            },
            "timeout": {
                "authState": "Identified",
                "uiState": "auth.success"
            }
        }
    },
    "productization": {
        "exampleA": 300,
        "exampleB": "notfoobar"
    }
}